client/rest/cmd.go

```diff
@@ -0,0 +1,45 @@
+package rest
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/spf13/cobra"
+
+	coinrest "github.com/tendermint/basecoin/modules/coin/rest"
+)
+
+var ServeCmd = &cobra.Command{
+	Use:   "serve",
+	Short: "Serve the light REST client for tendermint",
+	Long:  "Access basecoin via REST",
+	RunE:  serve,
+}
+
+var port int
+
+func init() {
+	ServeCmd.PersistentFlags().IntVar(&port, "port", 8998, "the port to run the server on")
+}
+
+const defaultAlgo = "ed25519"
+
+func serve(cmd *cobra.Command, args []string) error {
+	keysManager := DefaultKeysManager()
+	router := mux.NewRouter()
+	ctx := Context{
+		Keys: New(keysManager, defaultAlgo),
+	}
+	if err := ctx.RegisterHandlers(router); err != nil {
+		return err
+	}
+	if err := coinrest.RegisterHandlers(router); err != nil {
+		return err
+	}
+
+	addr := fmt.Sprintf(":%d", port)
```

Comment:
If you used Int() above, you could use `viper.GetInt(FlagPort)` here.
This would allow us to set a BC_PORT variable (12-factor style) or add `port = 12345` in the config file (typical unix style) to configure it and not just assume it comes from the command line.

Check out viper. It is pretty crazy.  Maybe overkill but we decided on it for the cli here, so we can go all out.